{"name": "Создание заказа с неверными хешами ингредиентов", "status": "failed", "statusDetails": {"message": "AssertionError: assert 500 == 400\n +  where 500 = <Response [500]>.status_code", "trace": "self = <test_create_order.TestCreateOrder object at 0x10307e1e0>\nauth_header = {'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY3NDdjZWRjOWVkMjgwMDAxYjUyNjdlNCIsImlhdCI6MTczMzI0NTA2MiwiZXhwIjoxNzMzMjQ2MjYyfQ.N3pO92zKz95cJU2WR6yAeBe4r5JWZZpsu8LDCqf6uk4'}\n\n    @allure.title(\"Создание заказа с неверными хешами ингредиентов\")\n    def test_create_order_with_invalid_ingredients(self, auth_header):\n        response = requests.post(\n            f\"{BASE_URL}{ENDPOINTS['orders']}\",\n            json={\"ingredients\": ORDER_DATA[\"invalid_ingredients\"]},\n            headers=auth_header\n        )\n>       assert response.status_code == 400\nE       assert 500 == 400\nE        +  where 500 = <Response [500]>.status_code\n\ntest_create_order.py:47: AssertionError"}, "start": 1733245062769, "stop": 1733245063079, "uuid": "a49ba9ec-b8f3-463a-a322-195074af9ebe", "historyId": "8da158c03f34cca19354eabf273e4086", "testCaseId": "8da158c03f34cca19354eabf273e4086", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_invalid_ingredients", "labels": [{"name": "feature", "value": "Создание заказа"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "96847-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}